app/Models/User.php

```diff
@@ -109,4 +109,11 @@ public function getAvatarAttribute()
     }
 
 
+    public function getNameAttribute()
+    {
+        return $this->profile->first_name . ' ' . $this->profile->last_name;
```

Comment:
Just code improvement in case no associated profile

public function getAvatarAttribute()
{
    return $this->profile ? $this->profile->avatar : null; // Return null or a default value if profile doesn't exist
}

public function getNameAttribute()
{
    if ($this->profile) {
        return $this->profile->first_name . ' ' . $this->profile->last_name;
    }
    return $this->name; // Fall back to the 'name' column from the User model or another default value
}

Reply:
I've edited the code! Please review it?